### DISCUSSION TITLE
app access doesn't work for me.

### DISCUSSION THREAD

**Discussion by Nisthar** [directus/directus#14613] [Archive]
I added a role for moderators with app access
added an existing a user into it, tried logging into the admin using the new user
i get this error

![](https://cdn.discordapp.com/attachments/1000835939813294171/1000836387144216686/unknown.png)

This is the indepth error when debugging

![](https://cdn.discordapp.com/attachments/1000835939813294171/1000835979868897340/unknown.png)

**Comment by rijkvanzanten** [comment]
Mind opening a new issue for this detailing the exact setup and steps to reproduce? Thanks!

**Reply by Nisthar** [reply]
> Mind opening a new issue for this detailing the exact setup and steps to reproduce? Thanks!

Yeah others doesn't seems to get this issue Right? otherwise there would be many issues on this. I will try updating the packages and try to pinpoint the issue.

**Reply by rijkvanzanten** [reply]
Something seems to go sideways in the particular configuration of the role, or the page you're trying to open. It's impossible to tell exactly what's going wrong with the current information presented though..

**Reply by Nisthar** [reply]
> Something seems to go sideways in the particular configuration of the role, or the page you're trying to open. It's impossible to tell exactly what's going wrong with the current information presented though..

I feel like this might be due to my permissions configuration. I enabled the "minimum app access config" for the role but it might be requesting something that the role doesn't have access to.

**Reply by Nisthar** [reply]
> Something seems to go sideways in the particular configuration of the role, or the page you're trying to open. It's impossible to tell exactly what's going wrong with the current information presented though..

Ok found the issue. Tried installing directus on a new directory. Connected the same database. It was the `users.read` hook that was causing it. Hopefully its fixed now.

**Comment by Nisthar** [comment]
Ok found the issue. Tried installing directus on a new directory. Connected the same database. It was the users.read hook that was causing it. Hopefully its fixed now.
